integration/exec_test.go

```diff
@@ -300,3 +303,78 @@ func TestEnter(t *testing.T) {
 		t.Fatal("The second process isn't in the required pid namespace", pidns, pidns2)
 	}
 }
+
+func TestFreeze(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+	root, err := newTestRoot()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(root)
+
+	rootfs, err := newRootFs()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer remove(rootfs)
+
+	config := newTemplateConfig(rootfs)
+
+	factory, err := libcontainer.New(root, []string{os.Args[0], "init", "--"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	container, err := factory.Create("test", config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer container.Destroy()
+
+	// Execute a first process in the container
```

Comment:
Take out this comment as it implies there might be a second process starting after?